code/api/src/setup/schema/queries.js

```diff
@@ -17,7 +18,8 @@ const query = new GraphQLObjectType({
     ...product,
     ...crate,
     ...subscription
+
   })
 })
```

Comment:
I remember an import of user_products here from the model, is that still necessary?

Reply:
The UserProducts has do queries so it shouldn't affect anything if it is deleted